lms/static/js/components/tabbed/views/tabbed_view.js

```diff
@@ -41,13 +66,21 @@
                        var self = this;
                        this.$el.html(this.template({}));
                        _.each(this.tabs, function(tabInfo, index) {
-                           var tabEl = $(_.template(tabTemplate, {
-                               index: index,
-                               title: tabInfo.title,
-                               url: tabInfo.url
-                           }));
+                           var tabId = tabInfo.url,
+                               tabEl = $(_.template(tabTemplate, {
+                                   index: index,
+                                   title: tabInfo.title,
+                                   url: tabInfo.url,
+                                   tabPanelId: tabId
```

Comment:
When do we expect `tabPanelId` and `tabId` to differ? Could we use the same template parameter for each? Could at least use `url: tabId`.